flo-freezer/pom.xml

```diff
@@ -26,5 +26,9 @@
       <groupId>com.twitter</groupId>
       <artifactId>chill-java</artifactId>
     </dependency>
+    <dependency>
+      <groupId>com.typesafe</groupId>
```

Comment:
This should be `optional` otherwise user code will pull in this dependency anyway and that makes `Class.forName()` meaningless. Is that right? I mean in fact flo-runner depends on typesafe config anyway, right? So maybe no need for `Class.forName()`.

Reply:
The reason for the `Class.forName()` is that `SimpleConfig` is a package private class, so I have not found any other way to reference it. I will add a comment about that.

About having the `config` dependency in `flo-freezer/pom.xml`, that is because the `Config` interface is references in the `ConfigSerializer` class.

Reply:
OK now I understood.